cli/resource/process.go

```diff
@@ -57,59 +84,58 @@ func ProcessFile(cli *cli.SensuCli, input string, recurse bool, processor Proces
 		if err != nil {
 			return err
 		}
-		resources, err := Parse(f)
+		resources, err = Parse(f)
```

Comment:
Is this not overwriting all the previous Parse, ultimately only keeping the last one?!
Unfortunately the test for `ProcessFile()` won't catch that because it uses a single file, in a single directory.

Reply:
Yes, I think this would break recursive processing.

Reply:
Confirmed, good catch.